omas/omas_imas.py

```diff
@@ -274,7 +274,8 @@ def imas_get(ids, path, skip_missing_nodes=False, check_empty=True):
 # save and load OMAS to IMAS
 # --------------------------------------------
 @codeparams_xml_save
-def save_omas_imas(ods, user=None, machine=None, pulse=None, run=None, occurrence={}, new=False, imas_version=None, verbose=True):
+def save_omas_imas(ods, user=None, machine=None, pulse=None, run=None, occurrence={}, 
+                   new=False, imas_version=None, verbose=True, backend='MDSPLUS'):
```

Comment:
Since you added a keyword, you should modify the docstring.

Reply:
Fixed in new commit